manifests/init.pp

```diff
@@ -335,11 +342,20 @@
     }
   }
 
-  package { $package_name_real:
-    ensure    => $package_ensure,
-    noop      => $package_noop_real,
-    source    => $package_source_real,
-    adminfile => $package_adminfile_real,
+  if $package_manage_real == true {
+    package { $package_name_real:
+      ensure    => $package_ensure,
+      noop      => $package_noop_real,
+      source    => $package_source_real,
+      adminfile => $package_adminfile_real,
+    }
+    $ntp_conf_require      = [ Package[$package_name_real] ]
```

Comment:
I don't think the variables need to be in the conditional logic. It should be achievable with something like

```puppet
if $package_manage_real == true {
  package { $package_name_real:
    ...
  }
}
```